Building python swig module fails due to either bad version written in version.mak or a bug in setup.py

### Describe the bug

upon cloning, and building with opus, and then trying to build python swig module i got this error

> Traceback (most recent call last):
>   File "/root/pjproject/pjsip-apps/src/swig/python/setup.py", line 106, in <module>
>     setup(name="pjsua2", 
>   File "/usr/local/lib/python3.10/dist-packages/setuptools/_distutils/core.py", line 146, in setup
>     _setup_distribution = dist = klass(attrs)
>   File "/usr/local/lib/python3.10/dist-packages/setuptools/dist.py", line 318, in __init__
>     self.metadata.version = self._normalize_version(self.metadata.version)
>   File "/usr/local/lib/python3.10/dist-packages/setuptools/dist.py", line 354, in _normalize_version
>     normalized = str(Version(version))
>   File "/usr/local/lib/python3.10/dist-packages/setuptools/_vendor/packaging/version.py", line 200, in __init__
>     raise InvalidVersion(f"Invalid version: '{version}'")
> setuptools.extern.packaging.version.InvalidVersion: Invalid version: '2.14--dev'
> make: *** [Makefile:42: _pjsua2.so] Error 1

so i modified the setup.py file that runs in the make process at line 59 to remove the concatenation of '-', then the building of swig worked

```
if pj_version_suffix:
     #pj_version += "-" + pj_version_suffix
     pj_version += pj_version_suffix 
```

not sure if the bug is in the setup.py or the version.mak file, but the version.mak file mentions to never be modified manually


### Steps to reproduce

1. clone master
2. build pjsip
3. build python swig module

### PJSIP version

2.14

### Context

building inside an ubuntu 20 container.

### Log, call stack, etc

```shell
N/A
```

i got this error when build python swig module 

```
/home/max/miniconda3/envs/pjsua2/compiler_compat/ld: /mnt/d/Coding/Application/pjproject-2.14.1/pjsip/lib/libpjsua2-x86_64-unknown-linux-gnu.a(media.o): warning: relocation against `_ZN2pj16AudioMediaPlayer6eof_cbEP12pjmedia_portPv' in read-only section `.text'
/home/max/miniconda3/envs/pjsua2/compiler_compat/ld: /mnt/d/Coding/Application/pjproject-2.14.1/pjsip/lib/libpjsua2-x86_64-unknown-linux-gnu.a(account.o): relocation R_X86_64_PC32 against symbol `_ZTVN2pj7AccountE' can not be used when making a shared object; recompile with -fPIC
/home/max/miniconda3/envs/pjsua2/compiler_compat/ld: final link failed: bad value
collect2: error: ld returned 1 exit status
error: command '/usr/bin/g++' failed with exit code 1
```

--- a/pjsip-apps/src/swig/python/setup.py
+++ b/pjsip-apps/src/swig/python/setup.py
@@ -63,7 +63,7 @@ pj_version = pj_version_major + "." + pj_version_minor
 if pj_version_rev:
     pj_version += "." + pj_version_rev
 if pj_version_suffix:
-    pj_version += "-" + pj_version_suffix
+    pj_version += pj_version_suffix
 
 #print 'PJ_VERSION = "'+ pj_version + '"'
 

